Products/ZenUI3/browser/search/resources/zenoss-search.js

```diff
@@ -21,34 +19,25 @@ Ext.onReady(function () {
          */
 
         var router = Zenoss.remote.SearchRouter;
-        var ManageSavedSearchDialog;
 
-        // Create two pickers to be swapped out.  (A picker is the drop down box)
-        var savedSearchPicker;
-        var freeSearchPicker;
         // Create two variables to hold the store ids
         var storedSearchesDataStoreID = "storedSearchesDataStoreID";
-        var  freeSearchDataStoreID = "freeSearchDataStoreID";
+        var freeSearchDataStoreID = "freeSearchDataStoreID";
 
         // Custom rendering Template for the Free Search Results
         // this defines the way the picker (drop down box) will look
         var resultTpl = new Ext.XTemplate(
             '<tpl for=".">',
-            '<table class="search-result"><tr class="x-combo-list-item">',
-            '<th><tpl if="values.category && (xindex == 1 || parent[xindex - 2].category != values.category)"',
-            '>{category}</tpl></th>',
-            '<td colspan="2" class="x-boundlist-item">{content}</td>',
+            '<tpl if="isSavedResult">',
+                '<div class="x-boundlist-item">{name}</div>',
+            '<tpl else>',
+                '<table class="search-result"><tr class="x-combo-list-item">',
+                '<th><tpl if="values.category && (xindex == 1 || parent[xindex - 2].category != values.category)"',
+                '>{category}</tpl></th>',
+                '<td colspan="2" class="x-boundlist-item">{content}</td>',
+            '</tpl>',
             '</tpl>');
 
-        // create an input field that everything hangs off of.
```

Comment:
remove this redundant field and wrap our search combo into container to make layout works right.